src/fsharp/FSharp.Core/result.fsi

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.FSharp.Core
+
+    open Microsoft.FSharp.Core.LanguagePrimitives.IntrinsicOperators
+
+    [<CompilationRepresentation(CompilationRepresentationFlags.ModuleSuffix)>]
+    module Result =
+
+        /// <summary><c>map f inp</c> evaluates to <c>match inp with Error e -> Error e | Ok x -> Ok (f x)</c>.</summary>
```

Comment:
These summaries should be English-language.  I know they're using the style from the "Option" summaries, but those ones should really be changed too :)

The MSDN docs use `summary strings like [`Transforms an option value by using a specified mapping function.``](https://msdn.microsoft.com/en-us/visualfsharpdocs/conceptual/option.map%5B't,'u%5D-function-%5Bfsharp%5D)